certbot-apache/certbot_apache/parser.py

```diff
@@ -291,33 +291,11 @@ def parse_modules(self):
 
     def update_runtime_variables(self):
         """Update Includes, Defines and Includes from httpd config dump data"""
-        self.update_defines()
+
+        self.variables = apache_util.parse_defines(self.configurator.option("ctl"))
```

Comment:
I think these should be consistent. If the `update` methods aren't used anywhere else, we can inline all of them, but otherwise I think we should keep `update_defines` around.

Reply:
Yeah, I agree. `update_defines()` is now reintroduced, even though as just as a wrapper method.

Reply:
Yeah, I agree. `update_defines()` is now reintroduced, even though it's just a wrapper method for the implementation above.